<launch>
        <!-- LAUNCH INTERFACE -->

        
        <!-- LAUNCH IMPLEMENTATION -->
        <!-- remember to set the parameters properly -->

        <node name="dual_manipulation_ik_control" pkg="dual_manipulation_ik_control" type="dual_manipulation_ik_control" respawn="false" output="screen"/> 
        <rosparam command="load" file="$(find dual_manipulation_ik_control)/config/ik_control.yaml" />

</launch>